### ISSUE TITLE
Sponsor SMPP support

### ISSUE THREAD

**Issue by ajamous** [sipcapture/heplify-server#494]
Hey Community, 

I would like to sponsor SMPP protocol v.3.4, 5.0 support for both capture agents, UI, and Grafana dashboard.  I think this would be beneficial to the sip capture community.

Anyone is interested to take this on?

**Comment by adubovikov** [comment]
@ajamous are you talking about SMS protocol ?

**Comment by adubovikov** [comment]
Closing wishlist without an active sponsorship/PR. Happy to reopen if someone funds or contributes SMPP capture end-to-end.

**Comment by ajamous** [comment]
> [@ajamous](https://github.com/ajamous) are you talking about SMS protocol ?

yes.

**Comment by adubovikov** [comment]
@ajamous thanks for confirming this is about SMS/SMPP.

SMPP is not implemented in heplify-server (or our capture agents) today. Before we can design anything, we need a real sample.

Please attach a **PCAP of SMPP traffic** (bind + `submit_sm` / `deliver_sm` and responses if available). If you know them, note SMPP version (3.4 vs 5.0) and whether the session is TLS.

We will review the capture and then decide next steps. Full stack (agent + UI + Grafana) stays out of scope until we have that sample.
